.github/workflows/update-containerd-version.yml

```diff
@@ -37,14 +37,30 @@ jobs:
           committer: minikube-bot <minikube-bot@google.com>
           author: minikube-bot <minikube-bot@google.com>
           branch: auto_bump_containerd_version
+          branch-suffix: short-commit-hash
           push-to-fork: minikube-bot/minikube
           base: master
           delete-branch: true
           title: 'bump containerd version'
-          labels: |
-            ok-to-build-iso
-            ok-to-build-image
           body: |
             containerd Project released a [new version](https://github.com/containerd/containerd/releases),
 
             This PR was auto-generated by `make update-containerd-version` using [update-containerd-version.yml](https://github.com/kubernetes/minikube/tree/master/.github/workflows/update-containerd-version.yml) CI Workflow.
+  build-comments:
+    needs: [
+      bump-containerd-version
+    ]
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/github-script@98814c53be79b1d30f795b907e553d8679345975
+        with:
+          github-token: ${{ secrets.MINIKUBE_BOT_PAT }}
+          script: |
+            github.rest.issues.createComment({
+              issue_number: ${{ steps.cpr.outputs.pull-request-number }},
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: |
```

Comment:
maybe it is better to write to separate comments ? to make jenkins not confused ? one for image one for iso